build/serve.js

```diff
@@ -116,6 +116,7 @@ gulp.task('spawn-backend', ['backend', 'kill-backend'], function() {
       [
         `--apiserver-host=http://${conf.backend.apiServerHost}`,
         `--port=${conf.backend.devServerPort}`,
+        `--heapster-host=http://${conf.backend.heapsterServerHost}`,
```

Comment:
I'm thinking about this a lot. The problem is that we want to talk with in-cluster heapster via service proxy. This is how clients would use. This is how my cluster works. This setup is only needed when you have our current dockerized cluster. 

Do you think it is possible to run heapster __inside__ the cluster? This would solve problems. 

If this is not possible, then we need to come up with a clever solution to make the Dashboard use in-cluster heapster   when available and external one when not available.